pkg/webhook/webhook.go

```diff
@@ -120,6 +121,7 @@ func (wh *Webhook) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 	if reviewResponse.Allowed == false {
 		if jsPolicy.Spec.AuditPolicy == nil || *jsPolicy.Spec.AuditPolicy != policyv1beta1.AuditPolicySkip {
 			go LogRequest(context.TODO(), wh.Client, req, reviewResponse, jsPolicy, wh.Scheme, 5)
+			go ReportRequest(context.TODO(), wh.Client, req, reviewResponse, jsPolicy, wh.Scheme, 5)
```

Comment:
please create an environment variable `ENABLE_POLICY_REPORTS` that needs to be `true` for this to get executed

Reply:
I added this ENV variable and a second one `POLICY_REPORT_MAX_EVENTS` to configure the max amount of events per report. I also mode this env vars configurable over the helm chart